oci/private/auth_config_locator.bzl

```diff
@@ -54,11 +51,11 @@ def _oci_auth_config_locator_impl(rctx):
             "\n",
             "Running one of `podman login`, `docker login`, `crane login` may help.",
         ], quiet = False)
-        rctx.file("config.json", "{}")
+        rctx.file("path", "")
     else:
-        rctx.symlink(config_path, "config.json")
+        rctx.file("path", config_path)
 
-    rctx.file("BUILD.bazel", """exports_files(["config.json"])""")
+    rctx.file("BUILD.bazel", """exports_files(["path"])""")
```

Comment:
"path" seems very short, what kind of path is it? If I'm poking around in the external/ folder I think a more explanatory name would be nice